### ISSUE TITLE
[Crash] Wyrms-of-Nyrus Crashed server

### ISSUE THREAD

**Issue by nenglizuida** [Vetpetmon-Labs/Wyrms-of-Nyrus-Hub#12]
### What happened?

It seems that when an alien event is triggered, the server tick times out and crashes?

The core is Cleanroom-0.2.3

### Version

0.7.x (Testing branch)

### Type/Severity

Crash to desktop/Server crash

### Relevant log output

```shell
https://mclo.gs/oCTU3GT
```

**Comment by Vetpetmon** [comment]
Oh yikes, I'd bring this up to the Cleanroom Loader developers, it seems that there's functions present that are either not in the newer JDKs (as indicated by NoSuchMethodError), or Cleanroom hasn't accounted for deep copying functions yet.

The source code for the devs there:
```java
public void duplicateThisEntity() {
        if (!this.world.isRemote && this != null) {
            try {
                final Class<? extends Entity> c = this.getClass();
                final Constructor<? extends Entity> constructor = c.getDeclaredConstructor(World.class);
                final Entity newInstance = constructor.newInstance(this.world);
                newInstance.deserializeNBT(this.serializeNBT());
                newInstance.setUniqueId(UUID.randomUUID());

                this.world.spawnEntity(newInstance);
            } catch (Throwable t) {
                t.printStackTrace();
            }
        }
    }
}
```

In the meantime, have you updated Cleanroom Launcher? My install script for the instance is a bit outdated and was made mainly for Meatballcraft.

**Comment by Vetpetmon** [comment]
The only entities with the Borgi variant are Soldiers. You can downgrade a version where the cloning functionality is not present. It looks like I may have to hard-code this after all...

**Comment by Vetpetmon** [comment]
Disregard everything I said above, it's nothing in deep cloning causing issues.

MOJANG, WHY IS `getLoadedEntityList()` CLIENT-SIDE ONLY???

**Comment by Vetpetmon** [comment]
Fixed indev, will close once the update is out on public release

**Comment by nenglizuida** [comment]
My integration is based on Cleanroom-0.2.3 and has only updated Fugue since then

**Comment by nenglizuida** [comment]
I don't really know much about mod making, I just read that the bug report points to Wyrms-of-Nyrus

**Comment by Vetpetmon** [comment]
This crash in specific is fixed in 0.7.75
